### PR TITLE
[exportdb] Setup website 

### PR DESCRIPTION
**Pull request by angelayi**
image

### CODE REVIEW

File: docs/source/scripts/exportdb/blurb.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ExportDB is a centralized dataset of supported and unsupported export cases.
+This is targeted towards users who want to understand specifically what types of
```
Comment (svekars) [inline docs/source/scripts/exportdb/blurb.md:2]:
```suggestion
It is targeted towards users who want to understand specifically what types of
```

File: docs/source/scripts/exportdb/blurb.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+code are supported, the subtleties of export, and how to modify their existing
+code to be compatible with export.  Note that this is not an exhaustive set of
```
Comment (svekars) [inline docs/source/scripts/exportdb/blurb.md:4]:
```suggestion
code to be compatible with export. Note that this is not an exhaustive set of
```

File: docs/source/scripts/exportdb/blurb.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+everything that is supported and unsupported by export, but will hopefully cover
```
Comment (svekars) [inline docs/source/scripts/exportdb/blurb.md:5]:
```suggestion
everything that is supported by exportdb, but it covers
```

File: docs/source/scripts/exportdb/blurb.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+most common and confusing use cases that users will run into.
+
+If you have a feature that you think needs a stronger guarantee from us to
+support in export please reach out to us!
```
Comment (svekars) [inline docs/source/scripts/exportdb/blurb.md:9]: how users can reach out? Create an issue with the correct tag in the pytorch/pytorch repo?

**Review by svekars — COMMENTED** [review]
A couple of editorial nits.

File: docs/source/scripts/exportdb/blurb.md (hunk 4 above)
Comment (angelayi) [reply docs/source/scripts/exportdb/blurb.md]: Thanks for the feedback! I updated it with asking users to tag module:export.

File: docs/source/scripts/exportdb/generate_example_rst.py [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+
+.. code-block:: python
+
+{splitted_source_code[1]}
+
+"""
+
+    # with open(os.path.join(EXPORTDB_SOURCE, f"{example_case.name}.rst"), "w") as f:
+    #     f.write(doc_contents)
+
```
Comment (zhxchen17) [inline docs/source/scripts/exportdb/generate_example_rst.py:98]: remove these?

File: docs/source/scripts/exportdb/generate_example_rst.py [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+    Generates the index.rst file
+    """
+
+    # support_level_to_modules = {}
+    # for _, example_case in sorted(example_cases.items()):
+    #     support_modules = support_level_to_modules.setdefault(
+    #         example_case.support_level, []
+    #     )
+    #     support_modules.append(example_case.name)
+
```
Comment (zhxchen17) [inline docs/source/scripts/exportdb/generate_example_rst.py:113]: extra code?

File: docs/source/scripts/exportdb/generate_example_rst.py [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+    for example_case in example_cases.values():
+
+        doc_contents = generate_example_rst(example_case)
+
+        for tag in example_case.tags:
+            tag_to_modules.setdefault(tag, []).append(doc_contents)
+
+        support_level_to_modules.setdefault(example_case.support_level, []).append(doc_contents)
+
+    generate_tag_rst(tag_to_modules)
```
Comment (zhxchen17) [inline docs/source/scripts/exportdb/generate_example_rst.py:179]: are tag rsts still being used?

File: docs/source/scripts/exportdb/generate_example_rst.py [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+        support_level = k.name.lower().replace("_", " ").title()
+        module_contents = "\n\n".join(v)
+        support_contents += f"""
+{support_level}
+{'-' * (len(support_level))}
+
+{module_contents}
+"""
+
+    tag_names = "\n    ".join(t for t in tag_to_modules.keys())
```
Comment (zhxchen17) [inline docs/source/scripts/exportdb/generate_example_rst.py:125]: I noticed there're "tag" appended to every tag name in the list, why is that? IMO better remove them.

File: docs/source/scripts/exportdb/generate_example_rst.py (hunk 7 above)
Comment (angelayi) [reply docs/source/scripts/exportdb/generate_example_rst.py:179]: yeah! theyre linked at the top

File: docs/source/scripts/exportdb/generate_example_rst.py (hunk 8 above)
Comment (angelayi) [reply docs/source/scripts/exportdb/generate_example_rst.py:125]: ok

File: docs/source/scripts/exportdb/generate_example_rst.py (hunk 7 above)
Comment (zhxchen17) [reply docs/source/scripts/exportdb/generate_example_rst.py:169]: is there still dedicated pages for them? or are they inlined?

Comment (angelayi) [reply docs/source/scripts/exportdb/generate_example_rst.py:169]: yeah, there are dedicated pages

**Review by zhxchen17 — APPROVED** [review]

**Pull request closed without merging**